File: 2212-removing-minimum-and-maximum-from-array/removing-minimum-and-maximum-from-array.cpp
class Solution {
public:
    int minimumDeletions(vector<int>& nums) {

        int mn = min_element(nums.begin(), nums.end()) - nums.begin();
        int mx = max_element(nums.begin(), nums.end()) - nums.begin();

        if (mn > mx) {
            swap(mn, mx);
        }

        int left = mx + 1;

        int right = nums.size() - mn;

        int both = (mn + 1) + (nums.size() - mx);

        return min({left, right, both});
    }
};

File: 2212-removing-minimum-and-maximum-from-array/removing-minimum-and-maximum-from-array.py
class Solution(object):
    def minimumDeletions(self, nums):
        n = len(nums)
        min_index = nums.index(min(nums))
        max_index = nums.index(max(nums))

        left = max(min_index, max_index) + 1
        right = n - min(min_index, max_index)
        both = (min(min_index, max_index) + 1) + \
               (n - max(min_index, max_index))

        return min(left, right, both)